<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Age&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Gender&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Type of Injury&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Blood Pressure&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Respiration Rate&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Eye mouvement&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Verbal response&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Motor coordination&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="typeOfInjuryComboBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Blunt      </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Penetrating</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="genderComboBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Female</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Male</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="ageSpinBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="bloodPressureSpinBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="respirationRateSpinBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="eyeMouvementComboBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>No Opening</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>To pain</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>To speech</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Spontaneous</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="verbalResponsecomboBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>No verbal response</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Incomprehensible sounds</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Inappropriate words</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Confused</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Oriented</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="motorCoordinationComboBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>No motor response</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Extension to pain</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Flexion to pain</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Withdrawal from pain</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Localising pain</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Obeys commands</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Abdomen&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Face&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Head&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Neck&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Thorax&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Spine&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Upper Extremity&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>230</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Lower Extremity&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="abdomenComboBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0 No injury</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 Minor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 Moderate</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 Serious</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4 Severe</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5 Critical</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6 Unsurvivable</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="faceComboBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0 No injury</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 Minor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 Moderate</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 Serious</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4 Severe</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5 Critical</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6 Unsurvivable</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="headComboBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0 No injury</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 Minor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 Moderate</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 Serious</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4 Severe</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5 Critical</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6 Unsurivable</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="neckComboBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>110</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0 No injury</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 Minor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 Moderate</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 Serious</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4 Severe</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5 Critical</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6 Unsurvivable</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="thoraxComboBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>140</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0 No injury</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 Minor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 Moderate</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 Serious</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4 Severe</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5 Critical</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6 Unsurvivable</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="spineComboBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>170</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0 No injury</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 Minor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 Moderate</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 Serious</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4 Severe</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5 Critical</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6 Unsurvivable</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="upperExtremityComboBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>200</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0 No injury</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 Minor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 Moderate</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 Serious</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4 Severe</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5 Critical</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6 Unsurvivable</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="lowerExtremityComboBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>230</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0 No injury</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 Minor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 Moderate</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 Serious</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4 Severe</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5 Critical</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6 Unsurvivable</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="predictPushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Predict</string>
    </property>
   </widget>
   <widget class="QLabel" name="probaLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>340</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#ff0000;&quot;&gt;probability&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newPatientPushButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>340</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>New Patient</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>260</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;External&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="externalComboBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>260</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0 No injury</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 Minor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 Moderate</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 Serious</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4 Severe</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5 Critical</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6 Unsurvivable</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="classLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#ff0000;&quot;&gt;class&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>161</width>
      <height>142</height>
     </rect>
    </property>
    <property name="title">
     <string>Method</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="id3RadioButton">
       <property name="text">
        <string>J48</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="randomForestRadioButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Random Forest</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="annRadioButton">
       <property name="text">
        <string>Neural Network</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="svmRadioButton">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>SVM</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="naiveBayesRadioButton">
       <property name="text">
        <string>Naive Bayes</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>310</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>310</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>536</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
